assets/training/model_management/src/azureml/model/mgmt/processors/pyfunc/vision/conda_is.yaml

```diff
@@ -0,0 +1,19 @@
+channels:
+- conda-forge
+dependencies:
+- python=3.9.19
+- pip<=24.0
+- pip:
```

Comment:
below is the list of vulnerabilities just for the first package https://[pypi.org/pypi/mlflow/2.12.1/json](https://pypi.org/pypi/mlflow/2.12.1/json)